src/Aquifer.Well.API/Endpoints/Languages/List/ListLanguagesEndpoint.cs

```diff
@@ -11,6 +11,11 @@ public override void Configure()
     {
         Get("/languages");
         Options(EndpointHelpers.ServerCacheInSeconds(EndpointHelpers.OneHourInSeconds));
+        Summary(s =>
+        {
+            s.Summary = "Return language list.";
+            s.Description = "Return a list of languages that can have associated resources in the Aquifer.";
+        });
```

Comment:
I'm ambivalent on if this documentation is valuable to add or not for the generated client.  I didn't turn on the redoc `/docs` page for the Well API but this description does show up in intellisense comments on the generated C# client types which may be useful.